core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -10659,4 +10659,46 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         .withSpark()
         .ok(expectedSparkSql);
   }
+
+  @Test public void newLineInLiteral() {
+    final String query = "SELECT 'netezza\n to bq'";
+    final String expectedBQSql = "SELECT 'netezza to bq'";
+    sql(query)
+        .withBigQuery()
+        .ok(expectedBQSql);
+  }
+
+  @Test public void newLineInWhereClauseLiteral() {
+    final String query = "SELECT *\n"
+        + "FROM \"foodmart\".\"employee\"\n"
+        + "WHERE \"first_name\" ='Maya\n"
+        + " Gutierrez'";
+    final String expectedBQSql = "SELECT *\nFROM foodmart.employee\n"
+        + "WHERE first_name = 'Maya Gutierrez'";
+    sql(query)
+        .withBigQuery()
+        .ok(expectedBQSql);
+  }
+
+  @Test public void literalWithBackslasheInSelectWithAlias() {
+    final String query = "SELECT 'No IBL' AS \"FIRST_NM\","
+        + " 'US\\' AS \"AB\", 'Y' AS \"IBL_FG\", 'IBL' AS "
+        + "\"PRSN_ORG_ROLE_CD\"";
+    final String expectedBQSql = "SELECT 'No IBL' AS FIRST_NM,"
+        + " 'US\\\\' AS AB, 'Y' AS IBL_FG,"
+        + " 'IBL' AS PRSN_ORG_ROLE_CD";
+    sql(query)
+        .withBigQuery()
+        .ok(expectedBQSql);
+  }
+
+  @Test public void literalWithBackslasheInSelectList() {
```

Comment:
just a spell mistake: add s after e in  `Backslashe`

Reply:
Done